.travis.yml

```diff
@@ -16,7 +16,7 @@ jobs:
   allow_failures:
     - env: DJANGO=master
   include:
-    - python: 3.6
+    - python: 3.8
```

Comment:
This is good, although running the QA test matrix against the earliest version provides better indication of support; versions are usually backwards compatible, but not forwards compatible. However, since we only lint and style check with Python 3.8 and earlier versions already have a good test coverage, I think this is quite suited for the project.